tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/Selenium/SeleniumCommon.cs

```diff
@@ -0,0 +1,167 @@
+// <copyright file="SeleniumCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using Datadog.Trace.Ci;
+using Datadog.Trace.Ci.Tags;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.Selenium;
+
+internal static class SeleniumCommon
+{
+    internal const string IntegrationName = nameof(IntegrationId.Selenium);
+    private const IntegrationId IntegrationId = Configuration.IntegrationId.Selenium;
+
+    internal const string CommandGet = "get";
+    internal const string CommandClose = "close";
+    internal const string CommandQuit = "quit";
+    private const string CookieName = "datadog-ci-visibility-test-execution-id";
```

Comment:
tiny nit, which _may_ already be out of scope depending on the spec/level of concern.

There's a maximum size to both headers and cookies, and I've seen internal apps hit this for various reasons. As this adds to an _existing_ header, we are increasing the size of that header, and _could_ cause an otherwise working app to fail. It might be worth trying to shorten the length of this cookie to avoid accidentally breaking apps?

Reply:
:/ This cookie is being read by the RUM library, a change like that require everyone involved (RUM and other tracer implementations). Let's leave it like that for now, and check if we hit the issue.

Reply:
> This cookie is being read by the RUM library

OK, that was the bit I wasn't sure about, no problem then 👍